processor/src/test/java/org/mapstruct/ap/test/javadoc/JavadocTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.javadoc;
+
+import org.assertj.core.api.AbstractCharSequenceAssert;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.mapstruct.ap.test.annotatewith.AnnotateWithEnum;
+import org.mapstruct.ap.testutil.IssueKey;
+import org.mapstruct.ap.testutil.ProcessorTest;
+import org.mapstruct.ap.testutil.WithClasses;
+import org.mapstruct.ap.testutil.runner.GeneratedSource;
+
+/**
+ * @author Jose Carlos Campanero Ortiz
+ */
+@IssueKey("2987")
+@WithClasses(AnnotateWithEnum.class)
```

Comment:
Why is this needed?

Reply:
Sorry, I probably copied and pasted the test skeleton from another class and I forgot to remove that code.